resources/js/Enums/TaskStatusEnum.ts

```diff
@@ -0,0 +1,7 @@
+enum TaskStatusEnum {
```

Comment:
So if we add a new status we have to do it in 2 places?

Reply:
yeah In this case we would need to change in both places.
We could make all the data coming from the backend tho. We kind of share the values and strings for it from the backend and frontend